infra/terraform/dev/deploy/task-definitions.tf

```diff
@@ -0,0 +1,178 @@
+module "cache" {
+  source = "github.com/cloudposse/terraform-aws-ecs-container-definition?ref=0.46.0"
+
+  container_name   = "cache"
+  container_image  = var.ecs_cache_image
+  container_memory = var.ecs_cache_memory
+  container_cpu    = var.ecs_cache_cpu
+  essential        = true
+
+  log_configuration = {
+    logDriver = "awslogs"
+    options = {
+      "awslogs-group"         = "ecs/cache-${var.environment}"
+      "awslogs-region"        = var.region
+      "awslogs-stream-prefix" = "ecs"
+    }
+  }
+}
+
+module "database" {
+  source = "github.com/cloudposse/terraform-aws-ecs-container-definition?ref=0.46.0"
+
+  container_name   = "database"
+  container_image  = "${data.aws_ssm_parameter.DB_ECR_URL.value}:develop"
+  container_memory = var.ecs_db_memory
+  container_cpu    = var.ecs_db_cpu
+  essential        = true
+
+  log_configuration = {
+    logDriver = "awslogs"
+    options = {
+      "awslogs-group"         = "ecs/database-${var.environment}"
+      "awslogs-region"        = var.region
+      "awslogs-stream-prefix" = "ecs"
+    }
+  }
+}
+
+module "php-fpm" {
+  source = "github.com/cloudposse/terraform-aws-ecs-container-definition?ref=0.46.0"
+
+  container_name   = "php-fpm"
+  container_image  = "${data.aws_ssm_parameter.WEB_ECR_URL.value}:develop"
+  container_memory = var.ecs_php_memory
+  container_cpu    = var.ecs_php_cpu
+  essential        = true
+
+  log_configuration = {
+    logDriver = "awslogs"
+    options = {
+      "awslogs-group"         = "ecs/php-fpm-${var.environment}"
+      "awslogs-region"        = var.region
+      "awslogs-stream-prefix" = "ecs"
+    }
+  }
+
+  links = ["cache:cache", "database:database"]
+
+  secrets = [
+    {
+      name      = "URL_DOMAIN"
+      valueFrom = "wikijump-dev-URL_DOMAIN"
+    },
+    {
+      name      = "URL_UPLOAD_DOMAIN"
+      valueFrom = "wikijump-dev-URL_UPLOAD_DOMAIN"
+    },
+    {
+      name      = "DB_HOST"
+      valueFrom = "wikijump-dev-DB_HOST"
+    }
+  ]
+
+  docker_labels = {
+    "traefik.enable"                                = "true"
+    "traefik.http.routers.php-fpm.rule"             = "Host(`${var.web_domain}`,`www.${var.web_domain}`,`${var.files_domain}`,`www.${var.files_domain}`)"
+    "traefik.http.routers.php-fpm.tls"              = "true"
+    "traefik.http.routers.php-fpm.tls.certresolver" = "mytlschallenge"
+  }
+
+  healthcheck = {
+    command     = ["CMD-SHELL", "curl -f http://localhost || exit 1"]
```

Comment:
`|| exit 1` is redundant, it's already outputting non-zero

Reply:
Fair. Their example for this worked and I left it alone. Fixed in next commit.